### PR TITLE
Add initial servlets and frontend set up

### PR DESCRIPTION
**Pull request by abbymapes**
Added basic interface and servlet folders

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -13,16 +13,11 @@
...
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+function startListening() {
+    const graphic = document.getElementById('conversation-container');
+    graphic.innerHTML = "<p style = 'color: #DB4437'>I am listening </p>";
+    //fetch("/textinput").then(response => response.json()).then((status) => {
```
Comment (dibenede) [inline portfolio/src/main/webapp/script.js:21]: remove commented out code

Comment (dibenede) [reply portfolio/src/main/webapp/script.js:21]: or do you want this now?

Comment (abbymapes) [reply portfolio/src/main/webapp/script.js]: removed

**Comment by abbymapes** [comment]
There should be new split up commits now (hopefully it worked)

**Comment by dibenede** [comment]
I think you basically got it. I think (but am not 100% sure) that the order is flipped though. When I look at it on your github project, it shows that fa00c4b was first and 2e70f6c was last assuming time is progressing going up. 

Anyways, I won't harp on that for this PR. You've done enough hoop jumping for one day :) 

LGTM after "Add frontend" renaming

**Pull request merged**